install.sh

```diff
@@ -65,6 +65,70 @@ echo "================= Installing Java 1.8.0 ==================="
 echo "================= Installing Ruby 2.5.0  ==================="
 /u16/ruby/install.sh
 
+echo "================= Installing Python packages ==================="
+apt-get install -y -q \
+  python-pip=8.1.1* \
+  python-software-properties=0.96.20.7 \
+  python-dev=2.7.12*
+
+pip install virtualenv
+
+echo "================= Adding JQ 1.5.1 ==================="
+apt-get install -y -q jq=1.5+dfsg*
```

Comment:
just call it 1.5 if possible. the minor versions keep breaking if we run jobs after few days